src/main/java/mezz/jei/render/ItemStackFastRenderer.java

```diff
@@ -58,56 +51,9 @@ private void uncheckedRenderItemAndEffectIntoGUI(IEditModeConfig editModeConfig,
 			return;
 		}
 
-		GlStateManager.pushMatrix();
-		{
-			GlStateManager.translatef(area.getX() + padding + 8.0f, area.getY() + padding + 8.0f, 150.0F);
-			GlStateManager.scalef(16F, -16F, 16F);
-			bakedModel = ForgeHooksClient.handleCameraTransforms(bakedModel, ItemCameraTransforms.TransformType.GUI, false);
-			GlStateManager.translatef(-0.5F, -0.5F, -0.5F);
-
-			Minecraft minecraft = Minecraft.getInstance();
-			ItemRenderer itemRenderer = minecraft.getItemRenderer();
-			itemRenderer.renderModel(bakedModel, itemStack);
-
-			if (itemStack.hasEffect()) {
-				renderEffect(bakedModel);
-			}
-		}
-		GlStateManager.popMatrix();
-	}
-
-	protected void renderEffect(IBakedModel model) {
```

Comment:
JEI optimizes rendering. It reduces the number of OpenGL calls by batching render stages (seen in `mezz.jei.render.IngredientListBatchRenderer#render`). This was done under the guidance of Fry.
Unless something changed in 1.15.1 (like Fry built these optimizations into vanilla), the code here is not exactly the same as `ItemRender.renderItemAndEffectIntoGUI` and this shouldn't be changed.

Reply:
That's why I made note of it, there isn't a simple way to call a straight itemRenderer.renderModel in 1.15 (that I've seen) and my few attempts were less then optimal, but my thoughts were this had been implemented in vanilla but I can look into it further.

Reply:
Ah I see now, no that doesn't look to have been implemented in vanilla I've just pushed a change up that re-implements the same functionality in 1.15.